On The Map/Controller/AddLocationViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  AddLocationVC.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/23/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import CoreLocation
+
+class AddLocationViewController: UIViewController {
+    
+    var coordinate: CLLocationCoordinate2D = CLLocationCoordinate2D()
+    var activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView()
+    
+    @IBOutlet weak var location: UITextField!
+    @IBOutlet weak var link: UITextField!
+    
+    @IBAction func cancelOut(_ sender: Any) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    @IBAction func addLocation(_ sender: Any) {
+        
+        activityIndicator.center = self.view.center
+        activityIndicator.hidesWhenStopped = true
+        activityIndicator.style = UIActivityIndicatorView.Style.gray
+        view.addSubview(activityIndicator)
```

Comment:
I would put this kind of initial view setup in `viewDidLoad` instead, and set it to `isHidden = true`, then unhide and start animating inside the IBAction here. Better separation of concerns that way, and that kind of initial setup is exactly what viewDidLoad is intended for.

Reply:
that makes sense